### DISCUSSION TITLE
Hexo-theme-async中文技术文档建议

### DISCUSSION THREAD

**Discussion by WYuHwn** [MaLuns/hexo-theme-async#123] [Ideas 奇思妙想]
佬，能不能在主题技术文档里面汇总一下front-matter的全部字段索引呢？特别是主题特有的😭

**Comment by MaLuns** [comment]
有哪些字段呢，主题特有的都在这里额 https://hexo-theme-async.imalun.com/guide/page.html#%E6%96%87%E7%AB%A0-posts

**Comment by WYuHwn** [comment]
噢噢 我的问题😭 有些是Hexo自带的，我自己没记清

WvHwn
________________________________
发件人: 白云苍狗 ***@***.***>
发送时间: Friday, May 31, 2024 12:53:54 PM
收件人: MaLuns/hexo-theme-async ***@***.***>
抄送: 山药 ***@***.***>; Author ***@***.***>
主题: Re: [MaLuns/hexo-theme-async] Hexo-theme-async中文技术文档建议 (Discussion #123)


有哪些字段呢，主题特有的都在这里额 https://hexo-theme-async.imalun.com/guide/page.html#%E6%96%87%E7%AB%A0-posts
